contracts/tasks/whitelist.ts

```diff
@@ -1,3 +1,5 @@
+import { task } from "hardhat/config";
```

Comment:
This script actually deprecated from our side.

Reply:
Just delete the file?

Reply:
hmm this is still a helpful script, I would keep it

Reply:
> hmm this is still a helpful script, I would keep it

If u wanna keep the script, should add import to fix lint error.